tartiflette/scalar/custom_scalar.py

```diff
@@ -10,7 +13,7 @@ def __init__(self, name: str, schema_name: str = "default"):
 
     def bake(self, schema):
         if not self._implementation:
-            raise Exception("No implementation given")
+            raise MissingImplementation("No implementation given")
```

Comment:
Can you add the scalar name in the message ? "No Implementation given for scalar < %s >" or something like that.